ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -304,7 +304,7 @@ tests:
   container:
     from: src
 - as: e2e-aws-ovn
-  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^images/(agent|alibabacloud|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
+  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^images/(baremetal|infrastructure-providers|installer|installer-altinfra|installer-artifacts|libvirt|openstack)|(^pkg/asset/agent)/)
```

I don't think we want to skip the e2e-aws-ovn job when there are changes to the installer images. We could skip it when there changes only to cluster-api providers other then aws.

---

agree, I still think we want:
```suggestion
  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|images|pkg(/.+)?|cmd/openshift-install|terraform/providers)/(agent|azure|azurestack|baremetal|gcp|ibmcloud|libvirt|nutanix|openstack|ovirt|powervs|vsphere)/)
```

That would ignore changes to only the baremetal and openstack images.

---

The only terraform providers left are "azurestack", "ibm", "ignition", "local", and "time". What about we simplify that logic to something like: 

```suggestion
  skip_if_only_changed: (^(docs|upi|hack)/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^images/(baremetal|libvirt|openstack))|(^terraform/providers)|(^cluster-api/providers/([^a]|a[^w]|aw[^s]).*/)|(/agent/)
```
it should still be the same effect but I find it easier to parse.